packages/storybook/stories/components/input/3_demo.stories.ts

```diff
@@ -17,7 +16,7 @@ const storyParams = {
   },
   type: {
     category: 'General',
-    defaultValue: ODS_INPUT_TYPE.text,
+    defaultValue: 'text',
```

Comment:
<img width="448" alt="Capture d’écran 2023-12-22 à 10 10 14" src="https://github.com/ovh/design-system/assets/53512103/8bb5119f-fe3a-4309-8d11-6a693b1abd12">
We should be able to select a type